src/views/dashboard/solution/ShowSolution.vue

```diff
@@ -4,39 +4,116 @@
     fluid
     tag="section"
   >
-    <p>Choose an execution to watch from the dropdown list:</p>
+    <base-v-component
+      heading="Show an execution"
+    />
     <v-autocomplete
       v-model="selectedInst"
       :items="instances"
       item-text="name"
       item-value="value"
+      filled
+      outlined
+      label="Choose an instance to watch from list"
     />
     <v-autocomplete
       v-model="selectedEx"
       :items="executions"
       item-text="name"
       item-value="value"
+      filled
+      outlined
+      label="Choose an execution to watch from list"
     />
-    <base-v-component
-      heading="Show an execution"
-    />
+    <v-row>
+      <v-col
+        cols="12"
+        sm="6"
+        lg="3"
+      >
+        <base-material-stats-card
+          color="info"
+          icon="mdi-trophy-outline"
+          title="Best solution"
+          :value="this.best_solution"
+        />
+      </v-col>
+
+      <v-col
+        cols="12"
+        sm="6"
+        lg="3"
+      >
+        <base-material-stats-card
+          color="red"
+          icon="mdi-poll"
+          title="Best bound"
+          :value="this.best_bound"
+        />
+      </v-col>
+
+      <v-col
+        cols="12"
+        sm="6"
+        lg="3"
+      >
+        <base-material-stats-card
+          color="success"
+          icon="mdi-slash-forward"
+          icon-height="199"
+          title="Gap (%)"
+          :value="this.gap"
+        />
+      </v-col>
+
+      <v-col
+        cols="12"
+        sm="6"
+        lg="3"
+      >
+        <base-material-stats-card
+          color="orange"
+          icon="mdi-timer-sand-empty"
+          title="Time (s)"
+          :value="this.time"
+        />
+      </v-col>
+    </v-row>
     <!-- <p>Original message: "{{ selectedLog }}"</p> -->
-    <progress-line-chart
+    <!-- <progress-line-chart
       :options="{ width: 800, height: 180 }"
       :dataset="dataset"
-    ></progress-line-chart>
+    ></progress-line-chart> -->
     <div class="py-3" />
   </v-container>
 </template>
 
 <script>
   import { mapState } from 'vuex'
   import { getInstance } from '@/api'
-  import ProgressLineChart from './LineGraph'
+  // import ProgressLineChart from './LineGraph'
   export default {
     name: 'ShowExecution',
     components: {
-      ProgressLineChart,
+      // ProgressLineChart,
+    },
+    props: {
+      best_solution: {
+        type: Number,
+        default: 0,
+      },
+      best_bound: {
+        type: Number,
+        default: 0,
+      },
+      gap: {
+        type: Number,
+        default: 0,
+      },
+      time: {
+        type: Number,
+        default: 0,
+      },
     },
```

Comment:
From what I see you're using these parameters to show them inside the app and you're filling them inside the component. if you're going to modify these parameters inside this component, you need to put them inside `data`, not in `props`. `props` is for "input arguments to the component" that are passed through the template in html. An example of `props`: 

https://github.com/baobabsoluciones/cornflow-admin/blob/93b198d3b851595ee35cebf5606726ded0ce4695/src/views/dashboard/main/ExecutionsTable.vue#L64-L71

that are filled here (in another file):

https://github.com/baobabsoluciones/cornflow-admin/blob/93b198d3b851595ee35cebf5606726ded0ce4695/src/views/dashboard/main/InstanceMenu.vue#L77-L81